test/integration/registration/addon_registration_test.go

```diff
@@ -395,8 +384,6 @@ var _ = ginkgo.Describe("Addon Registration", func() {
 		ginkgo.It("should register addon failed with invalid custom signer", func() {
```

Comment:
Is this ginkgo.It case necessary now? Should we remove it?

Reply:
removed